pkgs/tools/misc/ollama/patch/remove-git.patch

```diff
@@ -0,0 +1,71 @@
+--- a/llm/generate/gen_common.sh
++++ b/llm/generate/gen_common.sh
+@@ -44,49 +44,17 @@
+     fi
+ }
+ 
+-git_module_setup() {
+-    if [ -n "${OLLAMA_SKIP_PATCHING}" ]; then
+-        echo "Skipping submodule initialization"
+-        return
+-    fi
+-    # Make sure the tree is clean after the directory moves
+-    if [ -d "${LLAMACPP_DIR}/gguf" ]; then
+-        echo "Cleaning up old submodule"
+-        rm -rf ${LLAMACPP_DIR}
+-    fi
+-    git submodule init
+-    git submodule update --force ${LLAMACPP_DIR}
+-
+-}
++git_module_setup() { :; }
```

Comment:
Replacing the whole thing is brittle and will cause conflicts if upstream changes anything. How about setting `OLLAMA_SKIP_PATCHING` instead for the build, so the function doesn't run?